src/app/components/Amenities/Amenities.tsx

```diff
@@ -33,8 +35,8 @@ export default function Amenities() {
           <ul className={styles.flexWrapper}>
             {Amenities.map((item) => (
               <li className={styles.amenityItem} key={item}>
```

Comment:
@Dheeru12503 
you can make this a component if it's very small create this in this the same file.

Reply:
done

Reply:
@Dheeru12503 
see again "create this in this the same file."

Reply:
Done and Check it